### ISSUE TITLE
Angular 5 support

### ISSUE THREAD

**Issue by abdullaunais** [bonjurmrfirst/angular4-carousel#33]
Please add support for the new Angular 5.1

**Comment by heena22626** [comment]
Hey, 
Is it working for you with angular 5 now?

**Comment by orenAzuz** [comment]
Hey, Heena I'm having the same problem -> It's not working on angular 5 yet is there anything I can do about it by myself until you will fix this bug?

**Comment by abdullaunais** [comment]
@heena22626 No. currently no. I'm getting the same error that you have mentioned in #34. Thanks for the detailed error.
